skopt/optimizer/optimizer.py

```diff
@@ -172,7 +173,32 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                 "Expected acq_optimizer to be 'lbfgs' or 'sampling', "
                 "got %s" % acq_optimizer)
 
-    def ask(self):
+    def ask(self, n_points=None, strategy="CL"):
+        """Query point or multiple points to evaluate.
+
+        * `n_jobs` [int or None]:
+            Number of function evaluations that can be done in parallel.
+            If the value is None, a single point to evaluate is returned.
+            Otherwise a list of points to evaluate is returned of size n_jobs.
+        * `strategy` [string]:
+            Strategy to use in order to query multiple points to evaluate.
+            Is ignored if n_jobs = None.
+        """
+        if n_points is None:
+            return self._ask()
+
+        opt = copy.deepcopy(self)
+        opt.rng = self.rng  # set random generator to not generate same points
```

Comment:
You need not have to make a copy here. `rng` is a RandomState and not an integer. It will produce the same sequence of random numbers across runs and not the same random numbers is a single run.